src/app/cube/details/details.component.html

```diff
@@ -0,0 +1,20 @@
+<div *ngIf="learningObject" class="details-wrapper" [ngClass]="learningObject.length">
+  <cube-details-splash [learningObject]="learningObject"></cube-details-splash>
+  <div class="flex">
+    <div class="content">
+      <div class="description">
+        <div class="title">Description</div>
+        <p [innerHtml]="learningObject.goals[0].text"></p>
+      </div>
+      <cube-outcomes-detail-view [outcomes]="learningObject.outcomes"></cube-outcomes-detail-view>
+      <cube-children-detail-view *ngIf="learningObject.children && learningObject.children.length > 0" [length]="learningObject.length" [children]="learningObject.children"></cube-children-detail-view>
```

Comment:
Instead of using a conditional here, you should be able to check the values like this ... "learningObject?.length" and "learningObject?.children"